stl/src/winapinls.cpp

```diff
@@ -1,9 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-#include <yvals_core.h>
-
-#if _STL_WIN32_WINNT < _STL_WIN32_WINNT_VISTA
+#if !(defined(_M_ARM64) || defined(_CRT_APP) || defined(_ONECORE))

```

Comment:
Whereas I appreciate the headers and the make file cleanup, this is now more obscure.
Maybe comment? There are only two occurrences, and their number is not expected to grow.

Reply:
@cpplearner I tried my best to explain why ARM64, App, and OneCore are special and don't need these functions.